applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
And to be clear, cb_api_endpoints was only meant to augment/hint at what was found in the code; it was never meant to be the authority on it.